package.xml

```diff
@@ -1,17 +1,21 @@
 <?xml version="1.0"?>
-<package>
+<package format="2">
   <name>rslidar_sdk</name>
   <version>1.5.0</version>
   <description>The rslidar_sdk package</description>
   <maintainer email="ron.zheng@robosense.cn">robosense</maintainer>
```

Comment:
Add Trossen Robotics as a maintainer.

Reply:
Should other details like the license be the same?

Reply:
Yes, keep all other details the same. We can't change the license from the original anyways.